dll/win32/shell32/changenotify.cpp

```diff
@@ -9,6 +9,14 @@
 
 WINE_DEFAULT_DEBUG_CHANNEL(shcn);
 
+struct DWORD_ITEMIDLIST
+{
+    USHORT cb;
+    DWORD dwItem1;
+    DWORD dwItem2;
+    USHORT wTerminator;
+};
```

Comment:
The name is [SHChangeDWORDAsIDList](https://learn.microsoft.com/en-us/windows/win32/api/shlobj_core/ns-shlobj_core-shchangedwordasidlist) and the last member is called cbZero.

Reply:
Thanks.